Retrospective 26_02.05

Dabrytskaya opened the pull request.

Dabrytskaya committed:
Retrospective 26_02

yildiztugba commented on planning/retrospective.md:
```diff
@@ -35,4 +35,46 @@ This retrospectives will help our team to improve our collabration for the futur
+
+## Stop Doing
+
+- Save unformatted code before sending PR
```
These Two lines is wıth different format than others, can you check them?

And also it seems like week 1 and week 2 are different in format. Do you want to change?

Dabrytskaya committed:
Retro

Honestly i can`t understand how the format of 2 part can be different, because i made copy-past. But ok. I just re-save all.

yildiztugba reviewed (approved):
İ think it is not solved but i will approve and we can check after about what i said
